crates/lace-bin/Cargo.toml

```
@@ -0,0 +1,15 @@
+[package]
+name = "lace-bin"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+bumpalo = { version = "3.14.0", features = ["collections"] }
+clap = { version = "4.4.6", features = ["derive"] }
+logos = "0.13.0"
+miette = { version = "5.10.0", features = ["fancy"] }
+thiserror = "1.0.49"
+derive_more = "0.99.17"
+unionfind = {path="../unionfind"}
```

```suggestion
unionfind = { path="../unionfind" }
```
but more importantly, I think this can be
```suggestion
unionfind = { workspace = true }
```
(for all of them)